gui-libs/display-manager-init/files/display-manager-setup.initd-1

```diff
@@ -0,0 +1,13 @@
+#!/sbin/openrc-run
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+depend() {
+	need localmount
+}
+
+start() {
+	if get_bootparam "nox" ; then
+		touch /etc/.noxdm
```

Comment:
I think using /run/ would be a bit better here. The /run is clean on every boot and the one in /etc will stay.

Reply:
I did think about this but I've kept it for backwards compatibility.

A lot of the variables like these, I've not changed them for the same reason, similar to the boot parameters as well.

Should I do both?

Reply:
moved to only using `/run`